Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -138,12 +138,37 @@ YUI.add('ez-platformuiapp', function (Y) {
                     oldService.setNextViewServiceParameters(newService);
                 }
             });
-
+            if (this.get('routeConfig')) {
+                Y.Array.each(this.get('routes'), Y.bind(this._enrichRoute, this));
+            }
             // Listening for events fired on child views
             this.views.errorView.instance.addTarget(this);
         },
 
         /**
+         * Find the route configuration
+         *
+         * @protected
+         * @method _findRouteConfig
+         */
+        _findRouteConfig: function (routeName) {
+            return this.get('routeConfig')[routeName];
+
+        },
+
+        /**
+         * Enrich the route with the route configuration
+         *
+         * @protected
+         * @method _enrichRoute
+         */
+        _enrichRoute: function (route, index) {
+            if (this._findRouteConfig(route.name)) {
```

Comment:
the `_findRouteConfig` is not really necessary, you can directly check here `this.get('routeConfig')[route.name]`